edgelet/iotedge/src/check/checks/iotedged_version.rs

```diff
@@ -0,0 +1,92 @@
+use std::process::Command;
+
+use failure::{self, Context, Fail, ResultExt};
+use regex::Regex;
+
+use crate::check::{checker::Checker, Check, CheckResult};
+
+#[derive(Default, serde_derive::Serialize)]
+pub struct IotedgedVersion {
+    version: Option<String>,
+    //TODO: find if 1.* or specified
```

Comment:
What does this TODO mean?

Reply:
Css wants to know if the customer specified a specific version (1.0.8) or simply 1.*

Reply:
That has nothing to do with iotedged version. iotedged version is the version of the iotedged binary, which you can see is obtained by `iotedged --version`. The thing CSS is talking about the versions of edge agent and hub in the deployment template.

Reply:
(Still needs to be removed.)